docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,732 @@
+      options={options}
+      placeholder="Añade tus pronombres"
+      helperText="Elige hasta 2 grupos de pronombres para que aparezcan en tu perfil y otras personas sepan cómo referirse a ti. Puedes editarlos o eliminarlos en cualquier momento."
+    />
+  );
+}
+`}
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      description="ComboBox can be used as a controlled or an uncontrolled component. An uncontrolled ComboBox stores its own state internally and update it based on the user input. On the other side, a controlled ComboBox's state is managed by a parent component. The parent component's state passes new values through props to the controlled component which notifies changes through event callbacks."
+      title="Controlled vs Uncontrolled"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        title="Uncontrolled ComboBox"
+        description={`An uncontrolled ComboBox should be used for basic cases where no default value or tags are required. By passing \`options\` values to ComboBox, the component takes control and has fully manages its internal state. Don't use the \`inputValue\` or \`selectedOptions\` props, any value different from \`null\` and \`undefined\` activates a controlled ComboBox.`}
+        defaultCode={`
+function ComboBoxExample(props) {
+  const PRONOUNS = [
+    'ey / em',
+    'he / him',
+    'ne / nem',
+    'she / her',
+    'they / them',
+    've / ver',
+    'xe / xem',
+    'xie / xem',
+    'zie / zem',
+  ];
+
+  const options = PRONOUNS.map((pronoun, index) => ({ label: pronoun, value: 'value'+index }));
+
+  return (
+    <ComboBox
+      accessibilityClearButtonLabel="Clear the current value"
+      accessibilityShowButtonLabel="Show a list of pronoun values for this field"
+      label="Pronouns"
+      id="uncontrolled"
+      noResultText="No results for your selection"
+      options={options}
+      placeholder="Add your pronouns"
+      helperText="Choose your pronouns to appear on your profile so others know how to refer to you. You can edit or remove these any time."
+    />
+  );
+}
+`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        title="Controlled ComboBox"
+        description={` A controlled ComboBox is required if a selected value is set, as shown in the first example. In the second example, values are set programatically. Controlled Comboboxes with [tags](#Tags) are also controlled components. A controlled ComboBox requires three value props: \`options\`,  \`inputValue\`,  and \`selectedOptions\`. ComboBox notifies of changes via the \`onChange\`, \`onSelect\`, \`onBlur\`, \`onFocus\`, \`onKeyDown\`, and \`onClear\` props. All values displayed by ComboBox at any time are controlled externally. To clear \`inputValue\`, set the value to an empty string \`inputValue\` = \` "" \`, \`null\`  or \` undefined\` values turn ComboBox into an uncontrolled component.`}
```

Comment:
"ComboBox is notified of changes via ..."

Reply:
DONE